checkov/cdk/checks/python/S3BucketEncryption.yaml

```diff
@@ -1,20 +1,15 @@
 metadata:
-  version: 0.1
+  version: 0.2
   id: CKV_AWS_19
   name: Ensure all data stored in the S3 bucket is securely encrypted at rest
   category: ENCRYPTION
+  approach: define failing
 scope:
   languages:
     - python
 definition:
```

Comment:
why not use just:
```
definition:
  pattern: "aws_cdk.aws_s3.Bucket(<ANY>)"
  conditions:
    - not_pattern: "aws_cdk.aws_s3.Bucket(<ANY>, encryption=$ENC, <ANY>)"
    - or:
      - metavariable: $OPT
        pattern: "aws_cdk.aws_s3.BucketEncryption.UNENCRYPTED"
```

Reply:
That doesn't catch
```python
fail_2 = s3.Bucket(
            self,
            "example",
            encryption=s3.BucketEncryption.UNENCRYPTED,
        )
```

Updated to use metavariable